### ISSUE TITLE
Allow global arrays

### ISSUE THREAD

**Issue by 8BitPimp** [8BitPimp/ccml#6]
It would be nice to support global arrays:

```
var global_array[128]

function my_func()
  global_array[2] = 1234
end
```

**Comment by 8BitPimp** [comment]
Working on this with c23a18ac3cee0ad2f280786a168377a8bf3f79d1

**Comment by 8BitPimp** [comment]
Addressed in commit: c80472b63cfcb67fc40ad59334034f8333f3e450
